config/helm/chart/default/tests/Common/csi/clusterrole-csi_test.yaml

```diff
@@ -5,7 +5,7 @@ tests:
   - it: should not exist by default
```

Comment:
update the comment so its not confusing 😅

Reply:
(check other changes for the same 😉 )

Reply:
Fixed in c33e4906a5785055b85428cbefcc4a1a0474a556